### ISSUE TITLE
missing-timing warning token not supported

### ISSUE THREAD

**Issue by palemieux** [skynav/ttt#180]
`--no-warn-on missing-timing` fails with _Usage: invalid option argument: --no-warn-on: token 'missing-timing' is not a recognized warning token_

It looks like the `missing-timing` token is not defined at `defaultWarningSpecifications` in `TimedTextVerifier.java`

**Comment by skynavga** [comment]
The problem is that you were verifying with the ttml1 model. That warning token only applies when using the imsc1 model. See [1].

[1] https://github.com/skynav/ttt/blob/master/ttt-ttv/src/main/java/com/skynav/ttv/model/imsc/IMSC1.java#L399

**Comment by palemieux** [comment]
It failed with the following command line:

```
java -jar c:\\bin\\ttxv\\ttt-ttxv-3.0-SNAPSHOT-jar-with-dependencies.jar --model imsc1 --no-warn-on missing-timing Animation001.xml
Usage: invalid option argument: --no-warn-on: token 'missing-timing' is not a recognized warning token
```

**Comment by skynavga** [comment]
Please check the verifier output to see what model was actually used. You
may need to specify --force-model imsc1 if TTV can't actually determine it
is an IMSC1 document, e.g., due to no profile being present.

**Comment by palemieux** [comment]
```
java -jar c:\\bin\\ttxv\\ttt-ttxv-3.0-SNAPSHOT-jar-with-dependencies.jar --force-model imsc1 --no-warn-on missing-timing Animation001.xml
Usage: invalid option argument: --no-warn-on: token 'missing-timing' is not a recognized warning token
```

**Comment by skynavga** [comment]
I've checked further, and determined this is indeed a bug.

**Comment by skynavga** [comment]
Fixed in 9455a2bdf627e2e22e20b7b2b55805144ff85522.
